blackbox/docs/general/ddl/data-types.rst

```diff
@@ -316,6 +317,114 @@ converted to UTC without considering the time zone indication.
     date time types. That means date type columns must always be declared
     beforehand.
 
+.. _interval_data_type:
+
+Interval
+--------
+
+
+.. _interval-literal:
+
+Interval literal
```

Comment:
should be ok to keep the literal doc close to data type, we do it for some other data types as well, but for string literal we put it in lexical-stucture.